.github/workflows/build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: vendor/bin/rector --dry-run
 
       - name: Run test suite
-        run: bin/kahlan --config=kahlan-config-github-action.php --clover=clover.xml
+        run: php bin/kahlan --config=kahlan-config-github-action.php --clover=clover.xml
```

Comment:
this somehow needed for make run on windows CI , see 

https://github.com/kahlan/kahlan/actions/runs/13065903741/job/36458116070?pr=438#step:7:1